src/OSPSuite.SimModel/src/DESolver.cpp

```diff
@@ -214,7 +214,7 @@ namespace SimModelNative
 
 			//cache start execution time
 			//used to check if execution time limit exceeded (if applies)
-			double executionStartTime = clock() / CLOCKS_PER_SEC;
+			double executionStartTime = clock() / (double)CLOCKS_PER_SEC;
```

Comment:
this is actually done only for the tests. Otherwise returned `executionStartTime `and `currentTime `are integers and the difference `currentTime - executionStartTime` is zero for all simulations with run time < 1 second.